examples/physx-select.ts

```
@@ -47,55 +48,33 @@ class PanScript extends Script {
-  position: Vector3,
-  rotation: Quaternion
-): Entity {
+function addPlane(rootEntity: Entity, size: Vector2, position: Vector3, rotation: Quaternion): Entity {
   const mtl = new PBRMaterial(rootEntity.engine);
-  mtl.baseColor.set(
-    0.2179807202597362,
-    0.2939682161541871,
-    0.31177952549087604,
-    1
-  );
+  mtl.baseColor.set(0.2179807202597362, 0.2939682161541871, 0.31177952549087604, 1);
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 0.31177952549087606

(lint/correctness/noPrecisionLoss)